ui_dashboard_workbook/arc_os_performance/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+type: docs
+title: "Azure Arc-enabled servers OS Performance Workbook"
+linkTitle: "Azure Arc-enabled servers OS Performance Workbook"
+weight: 2
+description: >
+---
+
+## Azure Arc-enabled servers OS Performance Workbook
+
+This Jumpstart Drop provides an Azure Monitor workbook that is intended to provide a single pane of glass for monitoring Arc-enabled servers Operating System (OS) Performance. The Azure Monitor workbook acts as a flexible canvas for data analysis and visualization in the Azure portal, gathering information from several data sources and combining them into an integrated interactive experience.
+
+## Contributors
+
+This Jumpstart Drop was originally written by the following contributors:
+
+- [Alejandro Sánchez | Cloud Solution Architect at Microsoft](www.linkedin.com/in/asgsanchezgomez)
+
+## Prerequisites
+
+- Clone the Azure Arc Drops repository
+
+    ```shell
+    git clone https://github.com/Azure/arc_jumpstart_drops.git
+    ```
+
+- Log Analytics workspace with [VM Insights](https://learn.microsoft.com/azure/azure-arc/servers/learn/tutorial-enable-vm-insights#enable-vm-insights) data collected. Note that this workbook will work for any server (Arc-enabled or not) sending VM Insights data to a Log Analytics workspace.
```

Comment:
I would have the note separately as we do in jumpstart so it is more highlighted